src/app.js

```diff
@@ -27,28 +27,79 @@ export default class App extends React.Component {
 
   }
 
+  homepage(event) {
+    this.setState({page: 'users'}) // Trying to link back to page of users
+  }
+
   render() {
     if(this.state.page == 'login') {
       return(
           <div className = 'container'>
-            <h1>Connect As: </h1>
-            <div className="btn-group" role = "group">
-            <button type="button" className = "btn btn-secondary" id = "tbutton" onClick = { this.connect }>Tourist</button>
-            <button type="button" className = "btn btn-secondary" id = "gbutton" onClick = { this.connect }>Guide</button>
+            <div className="navbar navbar-default">
+              <div className="container-fluid">
+                <div className="navbar-header">
+                  <p id="logo">Wander</p>
+                </div>
+              </div>
+            </div>
+            <h2 id="title">Welcome to Wander! </h2>
+            <div className = 'col-sm-12 col-md-6'>
```

Comment:
The logo is clearly off center:

![screen shot 2016-09-25 at 3 01 23 pm](https://cloud.githubusercontent.com/assets/6868877/18817537/a7c45ba6-8330-11e6-8520-3efefce0ded9.png)